### DISCUSSION TITLE
LibreOffice convert says "Service Unavailable" when send a .docx file for pdf conversion

### DISCUSSION THREAD

**Discussion by VijayKumarVV** [gotenberg/gotenberg#555] [Q&A] [answered]
curl --location --request POST 'http://localhost:3000/forms/libreoffice/convert' \
--form 'files=@"/D:/burstIQ/ListOfDocs.docx"' \
--form 'nativePdfFormat="PDF/A-1a"'

[SYSTEM] api: server listening on port 3000
{"level":"error","ts":1669563816.496288,"logger":"api","msg":"convert to PDF: lock long-running LibreOffice listener: start long-running LibreOffice listener: LibreOffice listener socket not available: context deadline exceeded","trace":"4b348d6b-6e1c-4215-8561-17e421fe503d","remote_ip":"172.17.0.1","host":"localhost:3000","uri":"/forms/libreoffice/convert","method":"POST","path":"/forms/libreoffice/convert","referer":"","user_agent":"PostmanRuntime/7.28.0","status":503,"latency":10489922600,"latency_human":"10.4899226s","bytes_in":14785,"bytes_out":19}

**Comment by NiklasBr** [comment, marked as answer]
Might be #442 that you are seeing?

**Reply by VijayKumarVV** [reply]
Thanks @NiklasBr
